OpenEdXMobile/res/layout/fragment_course_combined_info.xml

```diff
@@ -1,39 +1,22 @@
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/content_error_root"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/white"
-    android:fillViewport="true"
-    tools:context="org.edx.mobile.view.CourseCombinedInfoFragment">
+    android:paddingLeft="@dimen/common_course_display_margin_side"
+    android:paddingRight="@dimen/common_course_display_margin_side">
 
-    <FrameLayout
-        android:id="@+id/content_error_root"
+    <org.edx.mobile.view.custom.EdxWebView
+        android:id="@+id/announcement_webview"
```

Comment:
`announcement_webview` should be renamed to webview to keep it in sync with app. There is nothing specially related to announcements in it.
Its corresponding variable name in fragment should be renamed as well.